ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -132,9 +132,9 @@ public QueuedCommandStatus enqueueCommand(final ConfiguredStatement<?> statement
       commandStatusMap.remove(commandId);
       throw new KsqlException(
           String.format(
-              "Could not write the statement '%s' into the "
+              "Could not write the statement into the "
                   + "command topic"
-                  + ".", statement.getStatementText()
+                  + "."
```

Comment:
Curious what's the reason `getStatementText` is removed here but replaced with `QueryMask.getMaskedStatement(statement.getStatementText()));` in `ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java`

Reply:
Right, I was different things but lets keep things simple with <retracted> on older branches -- changing StreamedQueryResource